osbs/build/build_request.py

```diff
@@ -235,6 +235,7 @@ def render_orchestrate_build(self):
             'source_registry_uri': self.spec.source_registry_uri.value,
             'sources_command': self.spec.sources_command.value,
             'vendor': self.spec.vendor.value,
+            'equal_labels': self.spec.equal_labels.value,
```

Comment:
This needs to be serialized back into a str. See how `smtp_error_addresses` is done for instance.
It might be a little more complex for `equal_labels` though.